charts/rhtap-dh/templates/extra-env.yaml

```diff
@@ -51,3 +51,10 @@ data:
     QUAY__API_TOKEN: {{ $quaySecretData.token }}
 {{- end }}
     QUAY__URL: {{ $quaySecretData.url }}
+{{- $jenkinsSecretObj := (lookup "v1" "Secret" .Release.Namespace "rhtap-jenkins-integration") }}
+{{- $jenkinsSecretData := ($jenkinsSecretObj.data | default dict) }}
+{{- if $jenkinsSecretData.token }}
+    JENKINS__BASE__URL: {{ $jenkinsSecretData.baseUrl }}
```

Comment:
JENKINS__BASE__URL -> JENKINS__BASEURL.

'_' is used to replace invalid characters.
'__' is used to separate nested fields.